xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2006,7 +2006,11 @@ void COutput::InitCycle()
         ||  method == VS_INTERLACEMETHOD_RENDER_BOB)
     {
         if (method == VS_INTERLACEMETHOD_AUTO)
+#if defined(HAVE_VPP)
```

Comment:
this marco does not make any sense nowadays. we don't compile Kodi in an environment where VPP is not present.